docs/branches.md

```diff
@@ -0,0 +1,31 @@
+## Release branches
+
+There are two release branches in Toltec, [stable](https://toltec.delab.re/stable) and [testing](https://toltec.delab.re/testing).
+As a user, **you always want to use the stable branch**, which is the default one when following the install instructions.
+The testing branch exists solely for the repository maintainers to make sure that packages work correctly before distributing them to users.
+It may contain packages that could cause breakage if you install them on your device.
+
+### Adding or updating a package
+
+New packages or package updates are exclusively proposed through pull requests based on the testing branch.
+A proposal can be merged into testing after review from a maintainer and if it builds successfully in the CI.
+If it is a proposal for a new package, the maintainer who reviews the pull request becomes the maintainer for that package.
+If it updates an existing package, the maintainer of that specific package should do the review.
+
+### Moving a package from testing to stable
+
+Each Monday, a pull request containing selected updates from the testing branch and targeting the stable branch can be opened.
+This pull request can only contain package changes dating from the previous Friday or older.
+Each of these package changes must be tested by a maintainer different from the maintainer of the affected package.
```

Comment:
Instead of only allowing to open a merge PR on Mondays with changes from the preceding Friday, we could open them on Saturdays and only merge starting from the next Monday. This would allow us to more easily review and test the changes over the week-end.

```suggestion
Each Saturday, a pull request can be opened with selected updates from the testing branch to merge in the stable branch.
This pull request can only be merged from the next Monday.
Each of these package changes must be tested by a maintainer different from the maintainer of the affected package.
```